src/hiccup/compiler.clj

```diff
@@ -100,12 +124,15 @@
   [[tag & content] merge-attributes-fn]
   (when (not (or (keyword? tag) (symbol? tag) (string? tag)))
     (throw (IllegalArgumentException. (str tag " is not a valid element name."))))
-  (let [[_ tag id class] (re-matches re-tag (util/as-str tag))
+  (let [[tag id class] (parse-tag (util/to-str tag))
```

Comment:
Make sure that let clauses are kept aligned. e.g. if there's a let clause like:

```clojure
(let [first  a
      second b]
```

Then if you change it, keep the alignment:

```clojure
(let [one    a
      second b]
```

Reply:
52d7bfb0a652851cea1adc796fc9d1320fabfcdc
37d66cda9d19581db42a4b0dcd9855d0030bd437